doc/classes/Tabs.xml

```diff
@@ -262,36 +262,51 @@
 	</constants>
 	<theme_items>
 		<theme_item name="button" type="StyleBox">
+			Background of the close button when it's being hovered with cursor.
 		</theme_item>
 		<theme_item name="button_pressed" type="StyleBox">
+			Background of the close button when it's being pressed.
 		</theme_item>
 		<theme_item name="close" type="Texture2D">
+			The icon for the close button (see [member tab_close_display_policy]).
 		</theme_item>
 		<theme_item name="decrement" type="Texture2D">
+			Icon for the left arrow that appears when there's too many tabs to fit them.
 		</theme_item>
 		<theme_item name="decrement_highlight" type="Texture2D">
+			Icon for the left arrow that appears when there's too many tabs to fit them, when it's impossible to click it.
```

Comment:
```suggestion
			Icon for the left arrow button that appears when there are too many tabs to fit in the container width. Used when the button is disabled (i.e. the first tab is selected).
```

This is backwards compared to TabContainer, did you confirm that this inconsistency is indeed how it works? I only peeked at `TabContainer`'s code to confirm its docs.

Reply:
I checked again and I actually made a mistake in both descriptions. The highlight icon is used when the mouse is hovered over the button. Disabled increment/decrement is hard-coded to be half-transparent .-.

Reply:
Ok, updated the descriptions in both TabContainer and Tabs.